test/pummel/test-crypto-timing-safe-equal-benchmarks.js

```diff
@@ -19,7 +19,7 @@ function runOneBenchmark(...args) {
   // Don't let the comparison function get cached. This avoid a timing
   // inconsistency due to V8 optimization where the function would take
   // less time when called with a specific set of parameters.
-  delete require.cache[require.resolve(BENCHMARK_FUNC_PATH)];
+  require.cache = Object.create(null);
```

Comment:
Can you add a comment here in case someone tries to go back to delete in the future?

Reply:
Sure, done.